src/models/datePickerFactory/DatePickerFactory.ts

```diff
@@ -26,59 +26,63 @@ export default class DatePickerFactory {
   private dateType: DateType;
   private maxDate?: Date;
   private minDate?: Date;
+  private seedDate: Date;
   private grouped?: boolean;
   dateTimeFormat: DateTimeFormat;
   private atomicDateObjectFunction: AtomicDateObjectCreator;
 
   constructor(config: ViewConfiguration) {
-    const { dateType, maxDate, minDate, grouped } = config;
+    const { dateType, maxDate, minDate, grouped, seedDate } = config;
```

Comment:
...but then should horizontal params get alphabetized too? I am conflicted.

Reply:
for a future refactor, when min/max are also in there :)